components/browser/menu/src/main/res/layout/mozac_browser_menu_item_image_text.xml

```diff
@@ -8,16 +8,22 @@
         style="@style/Mozac.Browser.Menu.Item.Container"
         android:layout_width="match_parent"
         android:orientation="horizontal"
-        android:gravity="center_vertical">
+        android:clickable="true"
+        android:gravity="center_vertical"
+        android:focusable="true">
 
     <android.support.v7.widget.AppCompatImageView
             android:id="@+id/image"
             style="@style/Mozac.Browser.Menu.Item.ImageText.Icon"
+            android:clickable="false"
+            android:background="@android:color/transparent"
             tools:src="@android:drawable/screen_background_dark"/>
 
     <TextView
             android:id="@+id/text"
             style="@style/Mozac.Browser.Menu.Item.ImageText.Label"
+            android:clickable="false"
+            android:background="@android:color/transparent"
```

Comment:
I'm surprised we need both; clickable and a transparent background? :o

Reply:
Clickable is for the click listener on the ViewGroup to take precedence and the transparent background is to disable the ripple animation. Oddly enough, disabling clickable doesn't mean the hover animation doesn't work. 🤷‍♂️